Orange/data/_argmaxrnd.pyx

```diff
@@ -0,0 +1,104 @@
+from hashlib import sha1
+from cython import boundscheck, cdivision
+import numpy as np
+cimport numpy as np
+from libc.stdlib cimport rand, srand, RAND_MAX, malloc, free
+
+
+cdef inline bint isnan(np.float_t x) nogil:
+    return x != x
+
+
+def argmaxrnd(np.ndarray vec not None, object random_seed=None):
+
+    """
+    Returns the index of the maximum value for a given 1D array.
+    In case of multiple indices corresponding to the maximum value,
+    the result is chosen randomly among those. The random number
+    generator, used by the C lang function 'rand', can be seeded
+    by forwarding a hashable python object. If -1 is passed, the
+    input array is hashed instead.
```

Comment:
Please add a comment that what this does is actually:
```py
np.random.choice((vec == np.nanmax(vec)).nonzero()[0])
```
where one can optionally pass a buffer-protocol-compliant object as random seed to seed the choice. Try to make it sound less trivial.

Please also explain in the comment **why** is _this_ used as opposed the line above
and **where**.